### DISCUSSION TITLE
Take our survey! Tell us what you think of github.com/community

### DISCUSSION THREAD

**Discussion by jricciardi** [community/community#33536] [Announcements]
Hello! 👋

We want to check in about your experience with github.com/community and invite you to provide thoughts and feedback on **[this survey](https://survey3.medallia.com/?4uGyzy-discussions-sep )**! How's it going?

We’ve been using [GitHub Discussions](https://github.com/features/discussions) to power github.com/community as the official platform for GitHub Q&A, show & tell, and feedback on GitHub products for [about two months now](https://github.com/orgs/community/discussions/27346).

What's working and where can we improve? Are you finding value here or getting your questions answered? What could take this experience up a notch? This 5-minute survey will help us better understand your experience. Your feedback will help inform our next iteration of this space, so please let us know what you think! We're excited to hear from you ✨

**[Link to survey](https://survey3.medallia.com/?4uGyzy-discussions-sep)**
This survey will close on October 4th.

Thank you!

**Comment by iLakshya** [comment]
I think that user experience should be the part to work on as it is a bit messed up (personal opinion). Rest everything is superb and you guys are doing a great job in making GitHub one of the best open-source platforms. Hats off to you guys.

**Comment by airtower-luna** [comment]
There seem to be some bugs in the survey… :sweat_smile: 

> How do you typically access github.com/community? Please select all that apply.

Can't actually select more than one there.

> If there is one thing you could change about github.com/community, what would it be?

That question occurs twice, the second time with a 0-10 scale. Maybe that was meant to be a different question?

**Reply by jricciardi** [reply]
These bugs appear to be resolved @airtower-luna - if you reload the survey and still see them, let us know!

**Comment by Gborw** [comment]
fix(vc): fix sth..... (#3306)
  ^         ^---------^  ^----^
  |         |            |
  |         +            +-> you issue id.
  |         |
  |         +-> Summary in present tense.
  |
  +-------> Type: feat, fix, docs, workflow, style, refactor, test, website, chore

**Comment by Gabri432** [comment]
### I would like the header bar to contain an option 'discussions' or 'community' to ease accessing the Github community discussions section.

I mean, the top bar that we have when opening any github webpage currently contains:
| search-bar | Pull requests option | Issues option | Marketplace option | Explore option |

I would like also to find a 'community' option in the top bar.

It might sound like I am too lazy for just taking more time to search the Community Discussions section but I struggle a bit to find it the first couple of times.

**Comment by AdityaArgadinata** [comment]
It is should be a good forum for discusion, a days ago my problem was solved from this

**Comment by Alvarezjorge82** [comment]
I'm exposing all you chumps

**Comment by CCurl** [comment]
I liked the dashboard before. Why change it? Now I have to learn this new thing, unless there is some setting to let us use that one instead.

Why can't people just leave well enough alone (not just you! :))?

**Comment by ibrahim-iqbal** [comment]
Hi,
one thing in regards to the survey, on the question "**How do you typically access the site**", it says Select all that apply, but clearly the options are radio buttons, and when selecting more than one the other gets deselected (expected behaviour from radio buttons)

![image](https://user-images.githubusercontent.com/41406463/193757028-298a6a25-b422-4f72-bfaa-ed61d25be158.png)

**Comment by hzh6567** [comment]
other



---Original---
From: ***@***.***&gt;
Date: Tue, Oct 4, 2022 15:12 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [community/community] Take our survey! Tell us what you think ofgithub.com/community (Discussion #33536)




 
Hi,
 one thing in regards to the survey, on the question "How do you typically access the site", it says Select all that apply, but clearly the options are radio buttons, and when selecting more than one the other gets deselected (expected behaviour from radio buttons)

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
